### PR TITLE
Adapt to API changes in LSP 1.28.0

### LINKED ISSUE #46: Unnecessary code?
I'm doing some refactoring in LSP and I was wondering why this code opens files after applying workspace edits. Doesn't `apply_parsed_workspace_edits` already opened them all?

https://github.com/sublimelsp/LSP-jdtls/blob/303f47bbf270f5df4251830a6b5c334659de4055/modules/test_extension_commands.py#L57-L66

### PR DESCRIPTION
**Pull request by rchl**
Adapt to changes (https://github.com/sublimelsp/LSP/pull/2393) in (yet to be released at the time of writing this) LSP 1.28.0.

Mainly updates the code that applies text edits plus some minor import changes to avoid "private" exports.

> [!WARNING]
> The changes need to be released at the same time as the new version of LSP

Resolves #46 (Haven't verified behavior before and after. Not sure if there are some minor behavior changes)

### CODE REVIEW

File: modules/test_extension_commands.py [hunk 1]
```diff
@@ -2,15 +2,14 @@
 import os
 
 import sublime
-from LSP.plugin import Session, parse_uri
+from LSP.plugin import Session, parse_uri, uri_from_view
```
Comment (predragnikolic) [inline modules/test_extension_commands.py:5]: ImportError: cannot import name uri_from_view

Comment (predragnikolic) [reply modules/test_extension_commands.py:5]: fixed with https://github.com/sublimelsp/LSP/pull/2393/commits/920a13c3c2e80391f91b16f11f005dfe74834e9c

**Review by predragnikolic — APPROVED** [review]

**Comment by predragnikolic** [comment]
I would wait for @LDAP to see this PR

File: modules/test_extension_commands.py [hunk 2]
```diff
@@ -52,17 +51,8 @@ def _on_done(workspace_edit: WorkspaceEdit):
                 print(workspace_edit)
                 return
 
-            parsed_worspace_edit = parse_workspace_edit(workspace_edit)
-
-            def open_changed_file(result):
-                window = self.view.window()
-                if window and parsed_worspace_edit:
-                    for uri in parsed_worspace_edit:
-                        open_and_focus_uri(window, uri)
```
Comment (LDAP) [inline modules/test_extension_commands.py:61]: Without this code, the test class isn't focused anymore.

**Review by LDAP — CHANGES_REQUESTED** [review]

File: modules/test_extension_commands.py (hunk 2 above)
Comment (rchl) [reply modules/test_extension_commands.py:61]: I don't have time to look into it now but do you have some simple way of reproducing (triggering this code)?

Comment (rchl) [reply modules/test_extension_commands.py:61]: I would like to see exactly what is happening here since it feels redundant on the first sight.

Comment (LDAP) [reply modules/test_extension_commands.py:61]: The difference is that this code does also focus the changed views.

See also https://github.com/sublimelsp/LSP/pull/2079#pullrequestreview-1135237911

**Comment by predragnikolic** [comment]
Just a small heads up I will soon create a LSP release and also a release for LSP-jdtls.

**Pull request merged**